pupil-spa/src/app/spoken-practice-question/spoken-practice-question.component.ts

```diff
@@ -24,6 +29,7 @@ export class SpokenPracticeQuestionComponent extends PracticeQuestionComponent i
 
   ngOnInit() {
     this.remainingTime = this.questionTimeoutSecs;
+    this.showPracticeBox = true;
```

Comment:
And this one?

Reply:
@jon-shipley Not needed any longer. Removed.

Reply:
@jon-shipley to clarify, I'm using existing var 'isWarmUp', there was no need to create a new variable so I removed 'showPracticeBox'.